### ISSUE TITLE
time inject node throws error

### ISSUE THREAD

**Issue by mivodu** [HM-RedMatic/node-red-contrib-sun-position#4]
After using the latest RedMatic version with latest node-red-contrib-sun-position I am getting the following error in the time inject node:

```
homematic-raspi daemon.err node-red: getFormattedDateOut date=[object Object] --> format=7  [time-comp.days] - [time-comp.month] [time-comp.dayDiffNames]
homematic-raspi daemon.err node-red[835]: [time-inject:3e2a72f0.04e0be] date.getTime is not a function
```

Time inject configuration:

![image](https://user-images.githubusercontent.com/47735196/52934076-ad0c1e00-3355-11e9-9991-1e296e2b4d24.png)

**Comment by Hypnos3** [comment]
I have checked, but I does not get this error. May be you have an other configuration.

Could you please select your time-inject node, [export](https://github.com/hm-redmatic/redmatic/wiki/Flow-Export) it and coy it here?

**Comment by mivodu** [comment]
`[{"id":"fc673fd0.076ee","type":"time-inject","z":"c5ce6edb.7a21","name":"","positionConfig":"8dc5e6c6.592b98","payload":"true","payloadType":"bool","payloadTimeFormat":"0","payloadOffset":"","payloadOffsetType":"none","payloadOffsetMultiplier":60000,"topic":"","time":"sunset","timeType":"pdsTime","timeDays":"*","offset":"","offsetType":"none","offsetMultiplier":60000,"property":"","propertyType":"none","timeAlt":"","timeAltType":"entered","timeAltOffset":0,"timeAltOffsetType":"none","timeAltOffsetMultiplier":60000,"once":false,"onceDelay":0.1,"addPayload1":"on_time","addPayload1Type":"msg","addPayload1Value":"23:00","addPayload1ValueType":"entered","addPayload1Format":"7","addPayload1Offset":"","addPayload1OffsetType":"none","addPayload1OffsetMultiplier":60000,"addPayload1Days":"*","addPayload2":"","addPayload2Type":"none","addPayload2Value":"23:00","addPayload2ValueType":"entered","addPayload2Format":"0","addPayload2Offset":"","addPayload2OffsetType":"none","addPayload2OffsetMultiplier":60000,"addPayload2Days":"*","addPayload3":"","addPayload3Type":"none","addPayload3Value":"","addPayload3ValueType":"date","addPayload3Format":"0","addPayload3Offset":"","addPayload3OffsetType":"none","addPayload3OffsetMultiplier":60000,"addPayload3Days":"*","recalcTime":2,"x":130,"y":100,"wires":[["e4484294.1f6a5","8ebc9dcb.6eba7"]]},{"id":"8dc5e6c6.592b98","type":"position-config","z":"","name":"Zuhause","isValide":"true","longitude":"0","latitude":"0","angleType":"deg","timezoneOffset":"1"}]`

**Comment by Hypnos3** [comment]
Sorry, I can't reproduce the specific problem in my test Environment with Version 0.2.2. (I remember, however, having seen and eliminated this bug in the earlier version).

Regardless of that, I found 2 other bugs.

Could you please install version 0.2.4 and check if your problem still exists with this Version?
(The version 0.2.4  should be available in a short time.)

**Comment by mivodu** [comment]
With 0.2.3 and as well with 0.2.4 the error in my system is gone. All working as expected. Thanks a lot for the fast support.

**Comment by Hypnos3** [comment]
seems to be fixed with 0.2.4
